tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -223,6 +275,33 @@ def do_log_copy(args: CensusBuildArgs) -> bool:
     return True
 
 
+def do_sync_release_file_to_replica_s3_bucket(args: CensusBuildArgs) -> bool:
+    """Copy release.json to replica S3 bucket"""
+    from .data_copy import sync_to_S3_remote
+
+    source_key = urlcat(args.config.cellxgene_census_S3_path, args.build_tag, "release.json")
+    dest_key = urlcat(args.config.cellxgene_census_S3_replica_path, args.build_tag, "release.json")
+
+    sync_to_S3_remote(
+        source_key,
+        dest_key,
+        dryrun=args.config.dryrun,
+    )
+    return True
+
+
+def do_sync_artifact_to_replica_s3_bucket(args: CensusBuildArgs) -> bool:
```

Comment:
I realize we have previously discussed this, but it would be MUCH better, if these `sync` operations had some sort of error checking that they are copying a clean, built, released Census, and not some random directory and/or failed build.

Reply:
I agree. I'll see what can be done, if it's too complex I'll file an issue to do this later, likely with the shared S3 build state.

Reply:
I think there are simpler checks that would be 80% of the value, for about 1% of the work (and can be replaced later if the better solution arrives).

Example:  just create a function that checks to see if the path "looks like" a Census build, e.g., contains well-known sub-paths, of nonzero length, etc.

No need to let this hold up the PR.